pkgs/by-name/bu/bugdom2/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bugdom2";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-+Snw5B5DPhzLPxKu9cThO/CNwoffNkSCLat1h1xRdAM=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    SDL2
+  ];
+
+  buildInputs = [
+    SDL2
```

Comment:
Seems like we should include libGL as well (and not only for darwin?)
See ofBorg failure on darwin: https://github.com/NixOS/nixpkgs/pull/279599/checks?check_run_id=21189764100